src/Controller/UI/AccountController.php

```diff
@@ -109,7 +109,12 @@ public function sendVerificationEmail(Request $request, MailSender $mailer)
         $emailAddress = $request->request->get('emailAddress');
         $reset = ($request->request->get('reset') == 'reset') ? true : false;
 
-        $people = $this->entityHandler->getBy(Person::class, array('emailAddress' => $emailAddress));
+        // Don't trust the public untrusted form to not send nulls/empties. EH doesn't like nulls.
```

Comment:
Please reword this comment.

Reply:
Yeah, that's giving a bit too much info in retrospect